src/router/index.js

```diff
@@ -33,12 +31,6 @@ const routes = [
     component: Home,
     meta: { requiresLogin: true }
   },
-  {
-    path: '/documents/:type',
```

Comment:
I guess this route is the most compelling argument _against_ removing this table, right?  We won't be able to easily support these views anymore without the old "documents" table.

I think I still want to go ahead.  If it turns out there was a compelling use case we're no longer serving, we can probably think up other ways to tackle it as well.

Actual feedback:

We should also remove the nav links associated with this route!

https://github.com/usdigitalresponse/arpa-reporter/blob/bbb9d4ac7c986f6e38c65266e82a0187a8625feb/src/components/Navigation.vue#L51-L71

Reply:
done